sim/rogue/garrote.go

```diff
@@ -42,19 +42,10 @@ func (rogue *Rogue) registerGarrote() {
 			NumberOfTicks: numTicks,
 			TickLength:    time.Second * 3,
 			OnSnapshot: func(sim *core.Simulation, target *core.Unit, dot *core.Dot, _ bool) {
-				dot.SnapshotBaseDamage = 119 + dot.Spell.MeleeAttackPower()*0.07
-				attackTable := dot.Spell.Unit.AttackTables[target.UnitIndex]
-				dot.SnapshotCritChance = dot.Spell.PhysicalCritChance(attackTable)
-				dot.SnapshotAttackerMultiplier = dot.Spell.AttackerDamageMultiplier(attackTable)
+				dot.Snapshot(target, 119+dot.Spell.MeleeAttackPower()*0.07)
```

Comment:
dot.Snapshot` will snapshot with `SpellCritChance`. I see the previous implementation used `PhysicalCritChance`. You probably want to expose a second `SnapshotPhysical` method that does it with the correct crit chance